### PR TITLE
CORE-15157 Library version update for `io.fabric8:kubernetes-client`

### PR DESCRIPTION
**Pull request by jacob-scott3141**

### CODE REVIEW

File: tools/plugins/preinstall/build.gradle [hunk 1]
```diff
@@ -25,7 +25,7 @@ dependencies {
     implementation "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:$jacksonVersion"
     implementation "com.fasterxml.jackson.core:jackson-databind:$jacksonVersion"
     implementation "com.fasterxml.jackson.module:jackson-module-kotlin:$jacksonVersion"
-    implementation "io.fabric8:kubernetes-client:6.5.1"
+    implementation "io.fabric8:kubernetes-client:6.7.2"
```
Comment (ronanbrowne) [inline tools/plugins/preinstall/build.gradle]: I know you're just bumping what was here already, but really this should be an external version like the others referenced from gradle.properties file

**Review by ronanbrowne — CHANGES_REQUESTED** [review]

**Review by davidcurrie — APPROVED** [review]
LGTM

**Review by seanbrereton — APPROVED** [review]
lgtm

**Review by yarivus — APPROVED** [review]
LGTM

**Review by ronanbrowne — APPROVED** [review]
LGTM

**Pull request merged**